projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -291,10 +291,10 @@ div.jp-carousel-buttons a:hover {
 	color: #999;
 	cursor: default;
 	letter-spacing: 0 !important;
-	padding: 0.35em 0 0;
+	padding: 0.35em 25px 0.35em 0;
```

Comment:
It looks like this might need a `box-sizing: border-box;` attribute added to ensure the padding is always within the `width: 100%` if the current theme doesn't set box-sizing globally.

Reply:
Great catch, thanks!